### PR TITLE
Fix whitespace in config

### PR DESCRIPTION
**Pull request by jkeiser**
No alphanumeric characters were harmed in the creation of this pull request.

### CODE REVIEW

File: omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb [hunk 1]
```diff
@@ -2,304 +2,330 @@
 %% -*- tab-width: 4;erlang-indent-level: 4;indent-tabs-mode: nil -*-
```
Comment (stevendanna) [inline omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb]: Are these magic comments wrong now?  Were they ever right?

Comment (jkeiser) [inline omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb]: Not really sure--most of this stuff wasn't indented correctly at all. I will go through and change all the indents to 4, though.

**Comment by jkeiser** [comment]
@stevendanna fixed to 4 spaces!

**Comment by stevendanna** [comment]
lgtm rebase and mwr

**Comment by jkeiser** [comment]
http://wilson.ci.chef.co/job/chef-server-12-trigger-ad_hoc/184/

**Pull request merged**